ansible/ansible-ai-connect-service
# AAP-24116: Enable only the appropriate Health check

Jira Issue: https://issues.redhat.com/browse/AAP-24116

## Description
This PR combines the explicit `WCA` and `WCAOnPrem` Health Checks.

It is only possible to have _one_ WCA model server at a given time; either "cloud" or "onprem".

See JIRA for example output scraped from an AAIC deployment using the service image generated by this PR.

## Testing
### Steps to test
1. Pull down the PR
2. Run `ansible-ai-connect-service` with WCA configuration.
3. Try with `ANSIBLE_AI_MODEL_MESH_API_TYPE` both `wca` and `wca-onprem` with applicable URL, Key, Model.
4. The Health Check should work in both scenarios (and use whatever WCA service has been configured)

### Scenarios tested
As above.

## Production deployment
